<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_average</class>
 <widget class="QDialog" name="dlg_average">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>平均价策略</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>中间价</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lb_middle_price">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>当前价格</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="sp_base_stock">
     <property name="maximum">
      <number>500000000</number>
     </property>
     <property name="value">
      <number>20000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>底仓</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="dbsp_max_price">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="value">
      <double>0.710000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>最高价</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="lb_target_benefit">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>目标盈利</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>最低价</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="dbsp_min_price">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="value">
      <double>0.450000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>10%涨幅</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>总涨幅</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lb_ten_pecent_increase">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QTextEdit" name="te_ladder"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>阶梯</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lb_total_increase">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>当前仓位</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pb_calculate_strategy">
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="dbsp_current_price">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="value">
      <double>0.450000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pb_calculate_trade_amount">
     <property name="text">
      <string>计算交易</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="lb_trade_amount">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>交易股数</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="sp_current_stock">
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
